org.eclipse.scanning.test/src/org/eclipse/scanning/test/malcolm/AbstractMultipleClientMalcolmTest.java

```diff
@@ -27,6 +27,9 @@
 import org.eclipse.scanning.api.malcolm.event.MalcolmEventBean;
 import org.junit.Test;
 
+/**
+ * TODO DAQ-1004 see comment in superclass
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)